runtime/near-test-contracts/src/lib.rs

```diff
@@ -49,12 +49,21 @@ pub fn rs_contract() -> &'static [u8] {
     include_bytes!(env!("CONTRACT_test_contract_rs"))
 }
 
-/// Standard test contract which is compatible any protocol version, including
-/// the oldest one.
+/// Standard test contract which is compatible with the latest supported
+/// protocol versions set.
```

Comment:
This wording is somewhat ambiguous IMO, and possibly incorrect? Wouldn't "latest supported protocol version" be the nightly protocol?

How about:

```suggestion
/// Standard test contract which is compatible with the earliest
/// protocol version still supported by current implementation of neard.
///
/// Note, that neard typically supports two or three protocol versions
/// at any given time.
```

would be clearer.

Reply:
good point, thanks for the suggestion!